docs/develop/java/overview.mdx

```diff
@@ -4,79 +4,100 @@ description: "Get an idea of what a Restate Java service looks like."
 ---
 
 import Admonition from '@theme/Admonition';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
 # Overview
 
-The Restate Java SDK is open source and can be found on GitHub: ([sdk-java repo](https://github.com/restatedev/sdk-java)).
+The Restate Java/Kotlin SDK is open source and can be found on GitHub: ([sdk-java repo](https://github.com/restatedev/sdk-java)).
 
 ## Installation
 
 <Admonition type="tip">
 Have a look at the [Java Quickstart](/get_started/quickstart?sdk=jvm)!
 </Admonition>
 
-Scaffold a project using the build tool of your choice. For example, with Gradle (Kotlin script):
+You can use the build tool of your choice with the Java/Kotlin SDK. The following instructions use Gradle (Kotlin script).
+
+<Tabs groupId="languages">
```

Comment:
Can we use this everywhere: `<Tabs groupId="sdk" queryString>`
This puts the group Id as a parameter in the URL which makes it possible to link specifically to the kotlin docs. And we use the groupId `sdk` everywhere else.